pkg/sql/distsqlrun/server.go

```diff
@@ -32,6 +32,38 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/tracing"
 )
 
+// MajorVersion and MinorVersion identify the distsqlrun protocol version.
+//
+// The planner populates major and minor versions in SetupFlorRequest.
+// A server only accepts requests with:
+//  - the same MajorVersion, and
+//  - a MinorVersion in the range MinAcceptedMinorVersion to MinorVersion.
+//
+// This can be used to provide a "window" of compatibility when new features are
+// added.
+//
+// Example:
```

Comment:
Forgot to update my comment: Is `MajorVersion` bumped on a backwards-incompatible change? I'm wondering if maybe it's better to simply have two `const`s that together provide a window of compatibility (`Version` and `MinSupportedVersion`) where `Version` and `MinSupportedVersion` are set to each other whenever there is a backwards-incompatible change.